app.py

```
@@ -176,7 +280,17 @@ async def auth_endpoint(body: TinodeRequest):
     email: str = claims.get("email", "")
     display_name: str = claims.get("name") or preferred
 
-    # Проверяем маппинг keycloak_id → tinode_uid
+    keycloak_id = userinfo.get("sub")
```

Ruff (0.15.12)

[error] Undefined name `userinfo`

(F821)